src/rust/en.mangairo/res/source.json

```
@@ -0,0 +1,25 @@
+{
+	"info": {
+		"id": "en.mangairo",
+		"lang": "en",
+		"name": "MangaIro",
+		"version": 1,
+		"url": "https://w.mangairo.com",
```

In order to get deeplinks to work
```suggestion
		"urls": [
			"https://w.mangairo.com",
			"https://chap.mangairo.com"
		],
```